docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/multidimensional-array.md

```diff
@@ -11,23 +11,19 @@ This check checks that project doesn't use the deeply nested multidimensional ar
 If a plugins stack is used on the project level, not all structures are necessarily required. Deeply nested multidimensional arrays make configuration hard to upgrade.
 This check verifies that multidimensional arrays have a maximum of two levels of nesting inside.
 
-## Example of an Evaluator error message
+## Example of an evaluator error message
 
 ```bash
 ======================
 MULTIDIMENSIONAL ARRAY
 ======================
 
-+---+----------------------------------------------------------------------------------------------------------------------------+------------------------------------------+
-| # | Message                                                                                                                    | Target                                   |
-+---+----------------------------------------------------------------------------------------------------------------------------+------------------------------------------+
-| 1 | Reached max level of nesting for the plugin registration in the {FormDependencyProvider::getPlugins()}.                    | Pyz\Yves\Module\ModuleDependencyProvider |
-|   | The maximum allowed nesting level is 2. Please, refactor code, otherwise it will cause upgradability issues in the future. |                                          |
-+---+----------------------------------------------------------------------------------------------------------------------------+------------------------------------------+
-
+Message: Reached max level of nesting for the plugin registration in the {FormDependencyProvider::getPlugins()}.
+         The maximum allowed nesting level is 2. Please, refactor code, otherwise it will cause upgradability issues in the future.
```

Comment:
Avoid predicting the future.
```suggestion
         The maximum allowed nesting level is 2. Refactor the code; otherwise, it can cause upgradability issues in the future.
```